cypress/support/pages/admin_console/Masthead.ts

```diff
@@ -50,9 +50,11 @@ export default class Masthead {
     cy.get("#manage-account").click();
   }
 
-  checkNotificationMessage(message: string) {
+  checkNotificationMessage(message: string, closeNotification?: boolean) {
```

Comment:
You can specify a default here as well.
```suggestion
  checkNotificationMessage(message: string, closeNotification = false) {
```